src/main/features/response/routes/reject-part-of-claim.ts

```diff
@@ -0,0 +1,53 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { FormValidator } from 'forms/validation/formValidator'
+import { Form } from 'forms/form'
+
+import { ResponseType } from 'response/form/models/responseType'
+import { RejectPartOfClaim } from 'response/form/models/rejectPartOfClaim'
+import { DraftService } from 'common/draft/draftService'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { GuardFactory } from 'response/guards/guardFactory'
+
+function isRequestAllowed (res: express.Response): boolean {
+  return res.locals.user.responseDraft.document.response !== undefined
```

Comment:
why don't we assign:

```
const document: responseDraft = res.locals.user.responseDraft.document
```

?

Reply:
We could, but I would like to create type definition more so that `res.locals.user.responseDraft.document` gives us right type. Having this we would not need to reassign to variable so often.